resources/translations/fi.edn

```diff
@@ -139,6 +143,7 @@
                                   :decision-requested "Pyydetty päätöstä"
                                   :draft-saved "Muokattu"
                                   :member-added "Lisätty jäsen"
+                                  :member-invited "Kutsuttu jäsen"
```

Comment:
Same here, as an event "Jäsen kutsuttu"? Although here the member-added is not in same form.

Reply:
In this case the other Finnish translation matches this form and I think both ways are fine so I'll keep it like this.